### ISSUE TITLE
custom options not work in promise mode for git.log

### ISSUE THREAD

**Issue by vfrbgt** [steveukx/git-js#464]
work correct
```javascript
async getCommits() {
    return new Promise((res, rej) => {
      this.gitInstance.log(["--all"], (err, result) => {
        if(err) {
          rej(err);
          return;
        }
        res(result);
      });
    });
  }
```

not apply "--all" option
```javascript
async getCommits() {
    let res = await this.gitInstance.log(["--all"]);
    return res;
  }
```

**Comment by steveukx** [comment]
Thank you for the detailed issue... this is resolved in version 2.7.0, available on npm now.

Based on your example, there is now a [unit test](https://github.com/steveukx/git-js/blob/master/test/unit/test-log.js#L200-L205) specifically covering the use of `await git.log(optionsArray)`

**Comment by dizel3d** [comment]
No, it doesn't work.
```
import simpleGit from 'simple-git';
const git = simpleGit(__dirname);
await git.log({ from, to, symmetric: true })
```

**Comment by steveukx** [comment]
Thanks - [another unit test](https://github.com/steveukx/git-js/blob/master/test/unit/test-log.js#L200-L219) and a fix for this was added in version 2.7.1 yesterday.
